clusters/hive/pools/msp/msp-ocp-4-8-small-simgle-az-amd64-aws-us-east-2_clusterpool.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: msp-ocp-4-8-small-simgle-az-amd64-aws-us-east-2
+  namespace: msp-cluster-pool
+  labels:
+    product: ocp
+    version: "4.8"
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    region: us-east-2
+    size: small
+    multi-az: false
+spec:
+  baseDomain: cpaas-ci.devcluster.openshift.com
+  imageSetRef:
+    name: ocp-4.8.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-2
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: cvp-aws-credentials
```

Comment:
I guess this secret does not exist in namespace `msp-cluster-pool`.

Reply:
That's correct, cvp-aws-credentials is only accesstible in cvp-cluster-pool namespace.